### PR TITLE
Minor import fixes

### PR DESCRIPTION
**Pull request by IViol**

### CODE REVIEW

File: src/components/WalletPasswordForm/walletPasswordForm.m.scss [hunk 1]
```diff
@@ -5,5 +5,11 @@
 
   position: relative;
   width: 544px;
+  height: 100%;
   margin: 0 auto;
 }
+
+.forgot {
+  position: absolute;
```
Comment (stcherenkov) [inline src/components/WalletPasswordForm/walletPasswordForm.m.scss]: Did you check it on mobile?

File: src/pages/WalletsImport/walletsImport.m.scss [hunk 2]
```diff
@@ -1,7 +1,10 @@
 @import 'styles/mixins';
 
 .core {
+  @include flex-row-center;
+
   width: 100%;
+  height: 100%;
```
Comment (stcherenkov) [inline src/pages/WalletsImport/walletsImport.m.scss]: Looks ok on narrow screens?

File: src/components/WalletPasswordForm/walletPasswordForm.m.scss (hunk 1 above)
Comment (IViol) [reply src/components/WalletPasswordForm/walletPasswordForm.m.scss]: just in chrome devices emulator

File: src/pages/WalletsImport/walletsImport.m.scss (hunk 2 above)
Comment (IViol) [reply src/pages/WalletsImport/walletsImport.m.scss]: Is there any place with etalon sketches for narrow screens? Which we can compare with our results?

Comment (stcherenkov) [reply src/pages/WalletsImport/walletsImport.m.scss]: No, we just check it for sanity right now. If it doesn't look bad or broken, it's fine

**Pull request merged**